systemd_ctypes/bus.py

```diff
@@ -242,7 +242,24 @@ class Bus(sd.bus):
     _default_user = None
 
     @staticmethod
-    def default_system(attach_event=False):
+    def new(fd=None, address=None, server=False, start=True, attach_event=True):
+        bus = Bus()
+        sd.bus.new(bus)
+        if address is not None:
+            bus.set_address(address)
+        if fd is not None:
+            bus.set_fd(fd, fd)
+        if server:
+            bus.set_server(True, sd.id128())
```

Comment:
We could use the `server` kwarg to pass the server id in: `bus.set_server(True, server)`.  And turn `True` into a random number maybe.

Reply:
I don't really care about this enough...  That's why I removed it.

the sd.id128 type is sort of awkward to use from outside.